### PR TITLE
Simplifying code

### PR DESCRIPTION
**Pull request by matiasinsaurralde**
Hi, these are some simplifications suggested by [gosimple](https://github.com/dominikh/go-simple).

Best.

### CODE REVIEW

File: host/chip/chip.go [hunk 1]
```diff
@@ -340,11 +340,8 @@ func (d *driver) Init() (bool, error) {
 		{gpioreg.ByName("CSID6"), gpioreg.ByName("CSID7")},
 		{U14_39, U14_40},
 	}
-	if err := pinreg.Register("U14", U14); err != nil {
-		return true, err
-	}
-
-	return true, nil
+	err := pinreg.Register("U14", U14)
```
Comment (maruel) [inline host/chip/chip.go]: `return true, pinreg.Register("U14", U14)`

File: host/distro/distro.go [hunk 2]
```diff
@@ -39,11 +39,11 @@ func IsArmbian() bool {
 func IsDebian() bool {
 	if isLinux {
 		// http://0pointer.de/public/systemd-man/os-release.html#ID_LIKE=
-		id, _ := OSRelease()["ID"]
+		id := OSRelease()["ID"]
 		if id == "debian" {
```
Comment (maruel) [inline host/distro/distro.go]: if OSRelease()["ID"] == "debian" {

File: host/distro/distro.go [hunk 3]
```diff
@@ -39,11 +39,11 @@ func IsArmbian() bool {
...
 			return true
 		}
-		idLike, _ := OSRelease()["ID_LIKE"]
+		idLike := OSRelease()["ID_LIKE"]
 		for _, part := range strings.Split(idLike, " ") {
```
Comment (maruel) [inline host/distro/distro.go]: `for _, part := range strings.Split(OSRelease()["ID_LIKE"], " ") {`
and same below.

File: host/sysfs/spi.go [hunk 4]
```diff
@@ -330,9 +330,7 @@ func (s *SPI) txInternal(w, r []byte) (int, error) {
 }
 
 func (s *SPI) setFlag(op uint, arg uint64) error {
-	if err := s.f.Ioctl(op|0x40000000, uintptr(unsafe.Pointer(&arg))); err != nil {
-		return err
-	}
+	err := s.f.Ioctl(op|0x40000000, uintptr(unsafe.Pointer(&arg)))
```
Comment (maruel) [inline host/sysfs/spi.go]: Revert this part, as otherwise it makes the commented out code unusable.

**Comment by maruel** [comment]
Thanks for the request! A few updates.

**Comment by matiasinsaurralde** [comment]
Done, no problem!

File: host/sysfs/spi.go [hunk 5]
```diff
@@ -344,7 +344,6 @@ func (s *SPI) setFlag(op uint, arg uint64) error {
 			return fmt.Errorf("sysfs-spi: op 0x%x: set 0x%x, read 0x%x", op, arg, actual)
 		}
 	*/
-	return nil
```
Comment (maruel) [inline host/sysfs/spi.go]: Add it back, it fails travis.

File: host/distro/distro.go [hunk 6]
```diff
@@ -58,7 +56,7 @@ func IsDebian() bool {
 // https://raspbian.org/
 func IsRaspbian() bool {
 	if isArm && isLinux {
-		id, _ := OSRelease()["ID"]
+		id := OSRelease()["ID"]
```
Comment (maruel) [inline host/distro/distro.go]: Inline these too.

**Comment by matiasinsaurralde** [comment]
👍

**Comment by maruel** [comment]
Technically the changes were mechanical but if you want to can add yourself to AUTHORS and CONTRIBUTORS. I'll leave you a few minutes to decide.

File: CONTRIBUTORS [hunk 7]
```diff
@@ -29,3 +29,4 @@
 Marc-Antoine Ruel <maruel@chromium.org> <maruel@gmail.com>
 Stephan Sperber <sperberstephan@googlemail.com>
 Thorsten von Eicken <tve@voneicken.com>
+Matias Insaurralde <matias@insaurral.de>
```
Comment (maruel) [inline CONTRIBUTORS]: Sorry for the nit picking but please keep the file with a trailing new line.

Comment (maruel) [inline CONTRIBUTORS]: And keep sorted.

Comment (matiasinsaurralde) [inline CONTRIBUTORS]: Sure!

**Comment by maruel** [comment]
Thanks!

**Pull request merged**